### PR TITLE
feat: filter transformer changes

### LINKED ISSUE #22: Transformers should be scoped to sourceId
This is currently working on startup, where the submissions view is queried with the correct start/endkey including `sourceId`.

But then the changes feed is queried without a filter, resulting in all transformers are run on all documents.

### PR DESCRIPTION
**Pull request by jo**
Add a filter to the transformer changes feed which filters the feed by pending
submissions of specified `sourceId`.

Closes #22

### CODE REVIEW

**Comment by jo** [comment]
needs https://github.com/eHealthAfrica/kazana/pull/66

File: lib/compose-plugin.js [hunk 1]
```diff
@@ -178,10 +178,16 @@ function registerTransformers (server, plugin, transformersMap, callback) {
     var indicatorName = tmp[0]
     var transform = tmp[1]
 
+    console.log('QUERY FOR CHANGES...', sourceId)
```
Comment (patriciagarcia) [inline lib/compose-plugin.js]: If I understood Kazana logging correctly, you can do `server.log` here

Comment (jo) [inline lib/compose-plugin.js]: Ups, this was committed by accident, thanks for pointing me at it :)

File: lib/compose-plugin.js [hunk 2]
```diff
@@ -178,10 +178,16 @@ function registerTransformers (server, plugin, transformersMap, callback) {
...
+
     server.app.admin.rawData.db.changes({
       since: 'now',
       live: true,
-      include_docs: true
+      include_docs: true,
+      filter: 'submissions/pending',
```
Comment (patriciagarcia) [inline lib/compose-plugin.js]: Not important but.. can this filter be used in line #197 also? Looks cleaner than the query with `startkey` and `endkey`.

Comment (jo) [inline lib/compose-plugin.js]: no, filters can only be used in changes feeds and replication.

Comment (patriciagarcia) [inline lib/compose-plugin.js]: ah, pity, it's good to merge, then :)

**Comment by patriciagarcia** [comment]
:+1:

**Pull request merged**